CONP_DATS_fields.md

```diff
@@ -1,47 +1,53 @@
+Derived From| |The sources this dataset is derived from.|The sources this dataset is derived from.|
+|Derived From|extraProperties - derivedFrom|**REQUIRED IF** | the dataset is a derived dataset, the URL of the dataset it is derived from is **REQUIRED**.  The original dataset must also be included as a submodule in the derived dataset.|The name of the source dataset this one is derived from.|
+|Parent dataset id|extraProperties - parent_dataset_id|**REQUIRE IF**| the dataset is a derived dataset, the parent dataset id (as specified in _conp_dataset/.gitmodules_) is **REQUIRED**.|The identifier (DOI) of the source dataset this one is derived from.|
+|Primary Publications|primaryPublications|**RECOMMENDED**| The primary publication(s) associated with the dataset, usually describing how the dataset was produced.| The primary publication(s) associated with the dataset, usually describing how the dataset was produced.|
+|Title|Primary Publications -> Title||The name of the publication, usually one sentence or short description of the publication.|The name of the publication.|
+|Publication Venue|Primary Publications -> Publication Venue||The name of the publication venue where the document is published (if applicable).|The name of the publication venue where the document is published (if applicable).|
+|Authors|Primary Publications -> Authors||Authors of the publication.|Authors of the publication.|
+|Affiliations|Primary Publications -> Authors -> Affiliations||Author affiliations.|Authors affiliations.|
+|Date|Primary Publications -> Dates||Relevant dates for the publication. If you provide a date, it must come with a description of the date.|Relevant dates for the publication. If you provide a date, it must come with a description of the date (i.e.: first submission, final approval, date of publication, ...).|
+|Identifier|Primary Publications -> Identifier|| Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the publication. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Identifier|Primary Publications -> Identifier -> Identifier|**RECOMMENDED**| A code uniquely identifying an entity locally to a system or globally.|A code uniquely identifying an entity locally to a system or globally.|
+|Identifier Source|Primary Publications -> Identifier -> Identifier Source|**RECOMMENDED**| Primary identifier for the publicationt. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the publication. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Dimensions|dimensions|**RECOMMENDED**| The different dimensions (granular components) making up a dataset. Providing dimensions give more details about the data types.|The different dimensions (granular components) making up a dataset. Providing dimensions give more details about the data types.|
```

Comment:
I don't understand what that is. @PapillonMcGill @emmetaobrien

Reply:
Dimensions refer to each of the variables collected in the study. (i.e.: handedness, weight, number of time event X append per day,...)

Reply:
I see. Might be worth giving examples there to clarify what is meant by dimension.

Reply:
excellent suggestion!